TheWarWithin/MageFire.lua

```diff
@@ -1467,6 +1505,14 @@ spec:RegisterAbilities( {
                 else
                     if buff.hot_streak.up then
                         removeBuff( "hot_streak" )
+                        if talent.spellfire_spheres.enabled then
+                            if buff.next_blast_spheres.stacks == 5 then
```

Comment:
Is this accounting for Rondumancy or whatever that changes the max_stacks?

Reply:
That's a different max stack counter.

next blast stacks to 5, then triggers and summons a sphere. The sphere stacks to 3 or 5 depending on rondurmancy, this 5 is static.